docs/root/configuration/upstream/cluster_manager/cluster_stats.rst

```diff
@@ -73,6 +73,7 @@ Every cluster has a statistics tree rooted at *cluster.<name>.* with the followi
   upstream_rq_rx_reset, Counter, Total requests that were reset remotely
   upstream_rq_tx_reset, Counter, Total requests that were reset locally
   upstream_rq_retry, Counter, Total request retries
+  upstream_rq_retry_budget_exceeded, Counter, Total requests not retried due to retry budgets
```

Comment:
I would just use the retry_overflow stat below. I don't think we need to add a new stat for this and waste the memory. I would be clear int he docs on how this can be incremented.